google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineAction.java

```diff
@@ -86,6 +86,7 @@ protected void executeProcess(
     commandLine.addParameter("--project=" + appEngineHelper.getProjectId());
     commandLine
         .addParameter("--credential-file-override=" + credentialsPath.getAbsolutePath());
+    commandLine.addParameter("--quiet");
     commandLine.withParentEnvironmentType(ParentEnvironmentType.CONSOLE);
     commandLine.getEnvironment().put("CLOUDSDK_METRICS_ENVIRONMENT", "gcloud-intellij");
     commandLine.getEnvironment().put("CLOUDSDK_APP_USE_GSUTIL", "0");
```

Comment:
A random question: what is the effect/purpose of CLOUSDK_APP_USE_GSUTIL=0? It seems related to service-side metrics.